tests/controllers/villains.test.js

```diff
@@ -0,0 +1,141 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const models = require('../../models')
+const {
+  after, afterEach, before, beforeEach, describe, it
+} = require('mocha')
+const { villainsList, singleVillain } = require('../mocks/villains')
+const { getAllVillains, getVillainBySlug, saveNewVillain } = require('../../controllers/villains.js')
+
+chai.use(sinonChai)
+const { expect } = chai
+
+describe('Controllers - villains', () =>
+{
+  let sandbox
+  let stubbedFindAll
+  let stubbedFindOne
+  let stubbedCreate
+  let stubbedSend
+  let response
+  let stubbedSendStatus
+  let stubbedStatusDotSend
+  let stubbedStatus
+
+  before(() =>
+  {
+    sandbox = sinon.createSandbox()
+    stubbedFindAll = sandbox.stub(models.villains, 'findAll')
+    stubbedFindOne = sandbox.stub(models.villains, 'findOne')
+    stubbedCreate = sandbox.stub(models.villains, 'create')
+    stubbedSend = sandbox.stub()
+    stubbedSendStatus = sandbox.stub()
+    stubbedStatusDotSend = sandbox.stub()
+    stubbedStatus = sandbox.stub()
+    response = {
+      send: stubbedSend,
+      sendStatus: stubbedSendStatus,
+      status: stubbedStatus,
+    }
+  })
+  beforeEach(() =>
+  {
+    stubbedStatus.returns({ send: stubbedStatusDotSend })
+  })
+  afterEach(() =>
+  {
+    sandbox.reset()
+  })
+  after(() =>
+  {
+    sandbox.restore()
+  })
+  describe('getAllVillains', () =>
+  {
+    it('Retrieves a list of villains and calls response.send() with the list.', async () =>
+    {
+      stubbedFindAll.returns(villainsList)
+      await getAllVillains({}, response)
+      expect(stubbedFindAll).to.have.callCount(1)
+      expect(stubbedFindAll).to.have.been.calledWith({ attributes: ['name', 'movie', 'slug'] })
+      expect(stubbedSend).to.have.been.calledWith(villainsList)
+    })
+    it('Returns a 500 error with an error message when the database call throws an error.', async () =>
+    {
+      stubbedFindAll.throws('ERROR!')
+      await getAllVillains({}, response)
+      expect(stubbedFindAll).to.have.callCount(1)
+      expect(stubbedStatus).to.have.been.calledWith(500)
+      expect(stubbedStatusDotSend).to.have.been.calledWith('Unable to retrieve all villains try again.')
+    })
+  })
+  describe('getVillainBySlug', () =>
+  {
+    it('Retrieves villain associated with the provided slug from the database and calls response.send ', async () =>
+    {
+      stubbedFindOne.returns(singleVillain)
+      const request = { params: { slug: 'hades' } }
+
+      await getVillainBySlug(request, response)
+      expect(stubbedFindOne).to.be.calledWith({
+        where: { slug: 'hades' },
+        attributes: ['name', 'movie', 'slug']
+      })
+      expect(stubbedStatus).to.have.been.calledWith(200)
+      expect(stubbedStatusDotSend).to.have.been.calledWith(singleVillain)
```

Comment:
You don't called `response.status(200).send()` in your controller. You just call `response.send()` with the villain returned from the database.